src/views/CommunityModeling.vue

```diff
@@ -284,7 +294,107 @@ export default Vue.extend({
     this.selectedMedium = this.media[0];
     this.selectedMethod = this.methods[0];
   },
+  mounted() {
+    this.renderCrossfeeding();
+  },
+  beforeDestroy() {
+    if (this.chart) {
+      this.chart.dispose();
+    }
+  },
   methods: {
+    cleanData(cross_feeding) {
+      if (cross_feeding.length > 0) {
+        let output = cross_feeding.map(obj => ({
+          ...obj,
+          to: this.getModelByID(obj.to).name,
+          from: this.getModelByID(obj.from).name
+        }));
+        return output;
+      } else {
+        return [];
+      }
+    },
+    renderCrossfeeding() {
+      let chart = am4core.create(this.$refs.chartdiv, am4charts.ChordDiagram);
+
+      chart.data = this.cleanData(this.communityData.cross_feeding);
+
+      chart.dataFields.fromName = "from";
+      chart.dataFields.toName = "to";
+      chart.dataFields.value = "value";
+      chart.sortBy = "value";
+      chart.responsive.enabled = true;
+
+      let slice = chart.nodes.template.slice;
+      slice.stroke = am4core.color("#000");
+      slice.strokeOpacity = 0.8;
+      slice.strokeWidth = 1;
+      slice.cornerRadius = 8;
+      slice.innerCornerRadius = 8;
+
+      var nodeTemplate = chart.nodes.template;
+      nodeTemplate.readerTitle = "Click to show/hide or drag to rearrange";
+      nodeTemplate.tooltipText = "{name}'s total exchanges: {total}";
+      nodeTemplate.propertyFields.fill = "color";
+
+      // Hovering over a Node highlights all connections that emmanate from that node.
+      nodeTemplate.events.on("over", function(event) {
```

Comment:
Minor: Prefer arrow functions when possible (there are multiple occurrences of this)